docs/privacy-policy.md

```diff
@@ -0,0 +1,55 @@
+---
+id: privacy-policy
+title: Privacy Policy
+---
+
+# Temporal Technologies, Inc. Privacy Policy
+
+Last updated: January 19th, 2021
+
+Temporal Technologies Inc. operates https://temporal.io (the "Site").
+This page informs you of our policies regarding the collection, use and disclosure of Personal Information we receive from users of the Site.
+
+We use your Personal Information only for providing and improving the Site.
+By using the Site, you agree to the collection and use of information in accordance with this policy.
+
+## Information Collection And Use
+
+While using our Site, we may ask you to provide us with certain personally identifiable information that can be used to contact or identify you.
+Personally identifiable information may include, but is not limited to your name ("Personal Information").
+
+## Log Data
+
+Like many site operators, we collect information that your browser sends whenever you visit our Site ("Log Data").
+This Log Data may include information such as your computer's Internet Protocol ("IP") address, browser type, browser version, the pages of our Site that you visit, the time and date of your visit, the time spent on those pages and other statistics.
+In addition, we may use third party services such as Google Analytics that collect, monitor and analyze this data.
+
+## Communications
+
+We may use your Personal Information to contact you with newsletters, marketing or promotional materials and other information that we believe is relevant for you.
```

Comment:
I think this is actually not super aligned with legal email subscription requirements. 
We can ONLY contact users who have explicitly subscribed to us. Other data collected can only be used for analytics.